control/trajectory_follower_node/src/controller_node.cpp

```diff
@@ -231,7 +233,8 @@ void Controller::callbackTimerControl()
   out.longitudinal = lon_out.control_cmd;
   control_cmd_pub_->publish(out);
 
-  // 6. publish debug marker
+  // 6. publish debug
```

Comment:
```suggestion
```

this comment line is unnecessary

Reply:
I couldn't understand here. The code owner was ordered the steps via comments so I did not want to delete this line.

**Original code:**
```
  // 4. sync with each other controllers
  longitudinal_controller_->sync(lat_out.sync_data);
  lateral_controller_->sync(lon_out.sync_data);

  // TODO(Horibe): Think specification. This comes from the old implementation.
  if (isTimeOut(lon_out, lat_out)) return;

  // 5. publish control command
  autoware_auto_control_msgs::msg::AckermannControlCommand out;
  out.stamp = this->now();
  out.lateral = lat_out.control_cmd;
  out.longitudinal = lon_out.control_cmd;
  control_cmd_pub_->publish(out);

  // 6. publish debug marker
  publishDebugMarker(*input_data, lat_out);
  
  ```

Reply:
fine 🙂‍↔️